sunpy/data/sample.py

```diff
@@ -11,7 +11,10 @@
 for _key in _sample_files:
     if os.path.isfile(os.path.join(_sampledata_dir, _sample_files[_key])):
         setattr(sys.modules[__name__], _key, os.path.join(_sampledata_dir, _sample_files[_key]))
+        print(os.path.join(_sampledata_dir, _sample_files[_key]))
     else:
         raise ImportError("Sample data file(s) missing. Use sunpy.data.download_sample_data() to get them.")
 
-__all__ = _sample_files.keys()
\ No newline at end of file
+file_list = _sample_files
```

Comment:
then why is it called `file_list` :p 

Also, I think it would be better if it were a list of filenames? At least from an iteration perspective, and allowing things like `sunpy.map.Map(sunpy.data.sample.file_list)`.